docs/2.6.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/participant.rst

```diff
@@ -1,142 +1,94 @@
-..
-     Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates
-..
-    
-..
-     Proprietary code. All rights reserved.
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
 
 .. enterprise-only::
 
 .. _ha_participant_arch:
 
-Replicated Participant Node Architecture
-----------------------------------------
+Replicating Participant Nodes
+-----------------------------
 
-High availability of a participant node is achieved with a replicated
-participant node in an active-passive configuration, where the active replica is
-serving requests and one or more passive replicas are in a warm stand-by mode
-ready to take over when the active replica fails.
+Participant nodes are replicated in an active-passive configuration with a shared database. 
+
+The active node services requests while one or more passive replicas wait in warm-standby mode, ready to take over if the active replica fails.
 
 High-Level System Design
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-A logical participant node can consist of multiple physical participant node replicas using a shared
-database and each replica exposing its own ledger API. However from an
-application point of view, the fact that multiple replicas exists can be hidden
-by exposing a single ledger API endpoint through a highly available load
-balancer.
+A logical participant node - shown below - contains multiple physical participant node replicas, all using a shared database. 
+
+Each replica exposes its own Ledger API although these can be hidden by a single Ledger API endpoint running on a highly available load balancer.
 
 .. _participant-ha-system-topology:
 .. https://lucid.app/lucidchart/cd96a3a6-e10b-4edc-bfb3-a70e484d7c06
 .. image:: participant-ha-system-topology.svg
    :align: center
    :width: 100%
 
-Why a Shared Database?
-""""""""""""""""""""""
+The load balancer configuration contains details of all Ledger API server addresses and the ports for the participant node replicas. Replicas expose their active or passive status via a health endpoint.
```

Comment:
this section is slightly out of date. now the ledger API server is shutdown when a node goes from active to passive, so the load balancer will also detect that the backend port becomes unreachable.